scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -0,0 +1,84 @@
+version 1.0
+
+import "GvsPrepareCallset.wdl" as GvsPrepareCallset
+import "GvsExtractCallset.wdl" as GvsExtractCallset
+
+# Workflow used by AoU to extract variants for a given cohort of sample_names
+
+workflow GvsExtractCohortFromSampleNames {
+
+  input {
+    File cohort_sample_names
+    String query_project
+    String gvs_project
+    String gvs_dataset
+    String fq_gvs_extraction_cohorts_dataset
+    String fq_gvs_extraction_destination_dataset
+    String fq_gvs_extraction_temp_tables_dataset
+    String extraction_uuid
+    String? output_gcs_dir
+
+    # Extract parameters
+    File wgs_intervals
+    Int scatter_count
+
+    File reference
+    File reference_index
+    File reference_dict
+
+    String output_file_base_name
+
+    Boolean do_not_filter_override = false
+    String? filter_set_name
+    String fq_filter_set_info_table = "~{gvs_project}.~{gvs_dataset}.filter_set_info"
+    String fq_filter_set_site_table = "~{gvs_project}.~{gvs_dataset}.filter_set_sites"
+    String fq_filter_set_tranches_table = "~{gvs_project}.~{gvs_dataset}.filter_set_tranches"
+
+    # if these are unset, default sensitivity levels will be used
+    Float? snps_truth_sensitivity_filter_level_override
+    Float? indels_truth_sensitivity_filter_level_override
+
+    File? gatk_override
+  }
+
+  call GvsPrepareCallset.GvsPrepareCallset {
+    input:
+      destination_cohort_table_prefix = extraction_uuid,
+      sample_names_to_extract         = cohort_sample_names,
+      data_project                    = query_project,
+      default_dataset                 = gvs_dataset, # unused if fq_* args are given
+      fq_petvet_dataset               = "~{gvs_project}.~{gvs_dataset}",
+      fq_sample_mapping_table         = "~{gvs_project}.~{gvs_dataset}.sample_info",
+      fq_temp_table_dataset           = fq_gvs_extraction_temp_tables_dataset,
+      fq_destination_dataset          = fq_gvs_extraction_destination_dataset
```

Comment:
maybe make these parameters in the definition of the task optional, since they are assigned defaults and then you won't have to pass the hard coded values

Reply:
so these do have defaults in `GvsPrepareCallset` but the default doesn't work for me which is why I overrode it here. is that what you're suggesting?

Reply:
ah, well then maybe add a comment that you are explicitly overriding the default values